src/transformers/models/transfo_xl/tokenization_transfo_xl.py

```diff
@@ -435,7 +439,8 @@ def moses_pipeline(self, text: str) -> List[str]:
 
     def _convert_id_to_token(self, idx):
         """Converts an id in a token (BPE) using the vocab."""
-        assert 0 <= idx < len(self), f"Index {idx} out of vocabulary range"
+        if 0 > idx or idx > len(self):
```

Comment:
```suggestion
        if 0 > idx or idx >= len(self):
```
The upper bound was strict before ;-)